### PR TITLE
Add camera sample

### PR DESCRIPTION
**Pull request by yne**

### CODE REVIEW

File: camera/src/main.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+				SceCameraRead read = {sizeof(SceCameraRead),0/*<Blocking*/};
+				sceCameraRead(cur_cam, &read);
+				sceDisplaySetFrameBuf(&dbuf, SCE_DISPLAY_SETBUF_NEXTFRAME);
+			}
+		}while(!(ctrl_press.buttons & (SCE_CTRL_UP|SCE_CTRL_DOWN|SCE_CTRL_LEFT|SCE_CTRL_RIGHT|SCE_CTRL_START)));
+		sceCameraStop(cur_cam);
+		sceCameraClose(cur_cam);
+		if(ctrl_press.buttons & SCE_CTRL_UP   )cur_res=(cur_res+1)%9;
+		if(ctrl_press.buttons & SCE_CTRL_DOWN )cur_cam=(cur_cam+1)%2;
+		if(ctrl_press.buttons & SCE_CTRL_LEFT )cur_fps=(cur_fps+1)%7;
```
Comment (xyzz) [inline camera/src/main.c]: use `ARRAYSIZE`-like macro here instead of hardcoding 9/7

**Comment by xyzz** [comment]
Also fix indentation please.

File: camera/src/main.c (hunk 1 above)
Comment (yne) [inline camera/src/main.c]: well spotted

**Comment by yne** [comment]
- ARRAYSIZE macro added 
- Fixed bad `if` indentation

File: camera/src/main.c [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+			.sizeIBase = 4*res_table[cur_res][0]*res_table[cur_res][1],
+			.pitch     = 4*(DISPLAY_WIDTH-res_table[cur_res][0]),
+			.pIBase = base,
+		});
+		sceCameraStart(cur_cam);
+		do{
+			ctrl_press = ctrl_peek;
+			sceCtrlPeekBufferPositive(0, &ctrl_peek, 1);
+			ctrl_press.buttons = ctrl_peek.buttons & ~ctrl_press.buttons;
+				sceCameraSetBrightness(cur_cam, ctrl_press.rx);
```
Comment (xyzz) [inline camera/src/main.c]: indent

Comment (yne) [inline camera/src/main.c]: Done

**Pull request merged**